### ISSUE TITLE
Implement view: facebook login

### ISSUE THREAD

**Issue by s0** [reach4help/reach4help#317]
* [x] Initial desktop implementation
* [x] Adapt implementation to mobile design

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/317)

*This information is updated automatically. To modify it, please use ZenHub.*

**Belonging to Epics:**

* **[EPIC]** [#312 - View: Landing Page](https://github.com/reach4help/reach4help/issues/312)

**Blocked By:**

* ~~[#317 - design view: login with facebook](https://github.com/reach4help/reach4help/issues/318)~~

**Blocking:**

* [BLOCKED BY: [#315](https://github.com/reach4help/reach4help/issues/315), [#316](https://github.com/reach4help/reach4help/issues/316), [#317](https://github.com/reach4help/reach4help/issues/317)] [[@ashwinkjoseph](https://github.com/ashwinkjoseph)] [#317 - Integrate view: landing page](https://github.com/reach4help/reach4help/issues/320)

**Comment by ashwinkjoseph** [comment]
is anyone currently working on this issue? @schowdhury8

**Comment by puzzledbytheweb** [comment]
This is already done I believe @ashwinkjoseph
